torchrl/objectives/returns/advantages.py

```diff
@@ -89,7 +91,7 @@ def __call__(
             if self.average_rewards:
                 reward = reward - reward.mean()
                 reward = reward / reward.std().clamp_min(1e-4)
-                tensordict.set_(
+                tensordict.set(
```

Comment:
`set_` is in-place and `set` not.
If we do in-place modifications with rewards that require gradients we can get graph errors. I was doing that to consume less memory but since rewards are scalars it should be fine.